docs/assets/templates/tools/barebones-.dockstore.yml

```diff
@@ -0,0 +1,10 @@
+version: 1.2
+tools:
+  - subclass: <CWL | WDL | NFL | GALAXY>
```

Comment:
the only subclass available for tools right now is CWL. WDL and NFL do not have the concept of tools and we will be following going forward. Galaxy does, but we don't have support for it.

Reply:
Oops, copy error on my part. Thanks for catching this.